### PR TITLE
sync zh-trans content/zh/docs/concepts/workloads/controllers/replicationcontroller.md

### PR DESCRIPTION
**Pull request by tanjunchen**
sync zh-trans content/zh/docs/concepts/workloads/controllers/replicationcontroller.md

### CODE REVIEW

File: content/zh/docs/concepts/workloads/controllers/replicationcontroller.md [hunk 1]
```diff
@@ -172,18 +178,26 @@ A ReplicationController also needs a [`.spec` section](https://git.k8s.io/commun
...
+### Pod 模板
 
+<!--
 The `.spec.template` is the only required field of the `.spec`.
 
 The `.spec.template` is a [pod template](/docs/concepts/workloads/pods/pod-overview/#pod-templates). It has exactly the same schema as a [pod](/docs/concepts/workloads/pods/pod/), except it is nested and does not have an `apiVersion` or `kind`.
+-->
+`.spec.template` 是 `.spec` 的唯一必需字段。
+
+`.spec.template` 是一个 [pod 模板](/docs/concepts/workloads/pods/pod-overview/#pod-templates)。它的模式与 [pod](/docs/concepts/workloads/pods/pod/) 完全相同，只是它是嵌套的，并且没有 `apiVersion` 或 `kind`。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/replicationcontroller.md]:
```suggestion
`.spec.template` 是一个 [pod 模板](/docs/concepts/workloads/pods/pod-overview/#pod-templates)。它的模式与 [pod](/docs/concepts/workloads/pods/pod/) 完全相同，只是它是嵌套的，没有 `apiVersion` 或 `kind` 属性。
```

File: content/zh/docs/concepts/workloads/controllers/replicationcontroller.md [hunk 2]
```diff
@@ -241,21 +262,10 @@ from doing this.
...
-ReplicationController 管理标签与选择器匹配的所有 Pods。
-它不区分它创建或删除的 pod 以及另一个人或进程创建或删除的 pod。
-这允许在不影响正在运行的 pod 的情况下替换 ReplicationController。
-
-如果指定了 `.spec.template.metadata.labels`，它必须和 `.spec.selector` 相同，否则它将被 API 拒绝。
-如果没有指定 `.spec.selector`，它将默认为 `.spec.template.metadata.labels`。
-
-此外，用户通常不应创建标签与此选择器匹配的任何其他 Pods，无论是直接创建、使用另一个 ReplicationController 来创建或者别的控制器（如 Job ）来创建，都是不允许的。
-如果这样做， ReplicationController 会认为这些 Pods 是由它自己创建的。
+另外，通常不应直接使用另一个 ReplicationController 或另一个控制器（例如 Job）来创建其标签与该选择器匹配的任何 Pod。如果这样做，ReplicationController 会认为它创建了其他 Pod。
```
Comment (tengqm) [inline content/zh/docs/concepts/workloads/controllers/replicationcontroller.md]:
```suggestion
另外，通常不应直接使用另一个 ReplicationController 或另一个控制器（例如 Job）来创建其标签与该选择器匹配的任何 Pod。如果这样做，ReplicationController 会认为它创建了这些 Pod。
```

**Comment by tanjunchen** [comment]
/cc @tengqm 
done

**Pull request merged**